docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -350,6 +297,38 @@ SELECT * FROM information_schema.tables;
 These pre-aggregations are stored as Parquet files under the `.cubestore/`
 folder in the project root during development.
 
+## Running in Production
+
+### Scaling
+
+Cube Store can be run in a single instance mode, but this is usually unsuitable
+for production deployments. For high concurrency and data throughput, we
+**strongly** recommend running Cube Store as a cluster of multiple instances
+instead.
+
+In cluster mode, Cube Store runs two kinds of nodes:
+
+- a single **router** node handles incoming client connections, manages database
+  metadata and serves simple queries.
+- multiple **worker** nodes which execute SQL queries
+
+In terms of configuration, both the router and worker nodes **must** specify the
+`CUBESTORE_SERVER_NAME` environment variable. Additionally, the router node
```

Comment:
The router must specify `CUBESTORE_META_PORT` too.

Reply:
Nice catch, thank you @ilya-biryukov 🙌